pythonforandroid/recipes/libzbar/__init__.py

```diff
@@ -17,7 +17,7 @@ class LibZBarRecipe(Recipe):
 
     def should_build(self, arch):
         return not os.path.exists(
-                os.path.join(self.ctx.get_libs_dir(arch.arch), 'libzbar.so'))
+            os.path.join(self.ctx.get_libs_dir(arch.arch), 'libzbar.so'))
```

Comment:
Minor E126 lint fix to demonstrate conditional recipe rebuild.
The `libzbar` recipe is currently `python2` only so the conditional build script should first try with current default target `python3` and then fallback to `python2`